sample_factory/enjoy.py

```diff
@@ -140,7 +151,12 @@ def max_frames_reached(frames):
                     if not cfg.no_render:
                         # TODO to render atari, need to add mode, will totally fix it in one week
                         # env.render(mode='rgb_array')
-                        env.render()
+                        if cfg.save_video:
```

Comment:
Can you move this to a separate function so that we don't repeat the same code twice?

Also I think we can remove @wmFrank 's comment about atari